### DISCUSSION TITLE
Differences from discord.py

### DISCUSSION THREAD

**Discussion by FeldrinH** [Pycord-Development/pycord#1471] [Q&A] [answered]
Both discord.py and pycord appear to be actively maintained.

Given that pycord is a fork of discord.py I would expect that the overall API and developer experience is similar. So which one should I use? What are the notable differences between pycord and discord.py? Will the libraries continue to diverge in the future?

**Comment by asterized** [comment, marked as answer]
first question: yeah the APIs are really similar, not much difference
second: it really depends. so far the slash command api (afaik) have been pretty similar, however discord.py is adopting some more lower level features
third: the full list of new dpy 2.0 features is here https://discordpy.readthedocs.io/en/latest/migrating.html
fourth: yes, the libraries will continue diverging (and pycord isn't planning on being archived any time soon)

**Comment by searchify7** [comment]
Use whatever you want
